### PR TITLE
Feature/documents mobile

### PR DESCRIPTION
**Pull request by Pettalves**

### CODE REVIEW

File: src/static/scss/components/_document-body.scss [hunk 1]
```diff
@@ -16,6 +16,11 @@
     margin-right: auto;
     margin-left: 3rem;
     padding-right: 3rem;
+
+    @include bp-smallonly {
+      margin-left: 0;
```
Comment (cfviotti) [inline src/static/scss/components/_document-body.scss:36]: I also would suggest maybe removing this right padding along with the left padding on the sidebar wrapper, and resorting only to the outer document padding itself for spacing. A bit more simpler that way

**Review by cfviotti — CHANGES_REQUESTED** [review]
- The sidebar (which really isn't a "side"bar, maybe renaming that to something more generic would be nice) does not extend its height fully when on mobile
image

- It also wont close itself when you resize your window back to a wider width.
I would suggest solving this by styling the sidebar itself to become somewhat like a modal on mobile, instead of duplicating its content in the markup. It should be possible.
image

- The close button is not doing anything right now.

- Right before the mobile breakpoint the content seems too constricted. I guess adjusting the tablet breakpoint with narrower paddings would suffice.
image

- Lastly please make sure all tests are passing

File: src/static/scss/components/_document-body.scss (hunk 1 above)
Comment (Pettalves) [reply src/static/scss/components/_document-body.scss:36]: Fixed

**Comment by Pettalves** [comment]
> * The sidebar (which really isn't a "side"bar, maybe renaming that to something more generic would be nice) does not extend its height fully when on mobile
>   image
> * It also wont close itself when you resize your window back to a wider width.
>   I would suggest solving this by styling the sidebar itself to become somewhat like a modal on mobile, instead of duplicating its content in the markup. It should be possible.
>   image
> * The close button is not doing anything right now.
> * Right before the mobile breakpoint the content seems too constricted. I guess adjusting the tablet breakpoint with narrower paddings would suffice.
>   image
> * Lastly please make sure all tests are passing

Fixed

**Review by cfviotti — CHANGES_REQUESTED** [review]
- Please make sure the document text is centralized when on a table breakpoint. It looks odd.
image

- Remove this margin as it makes the document text uncentered on mobile
image

- Make sure the body has its scroll locked when the modal is open
image

**Review by cfviotti — APPROVED** [review]

**Pull request merged**